en/MDN_Plus_Terms

```diff
@@ -0,0 +1,71 @@
+# MDN Plus  Terms of Service
+
+Version 1.0, effective March 9, 2022
+{: datetime="2022-03-09" }
+
+MDN Plus is a subscription service providing premium functionality, including MDN offline, bookmarking, themes, and notifications. (the “Service”). Please read these Terms of Service carefully because they explain important information about your use of the Service.
+
+## You Must Be Eligible to Use the Service
+
+You must be in a country where the Service is available. Currently, the Service is available in the United States, Canada, the United Kingdom, Germany, France, Italy, Spain, Belgium, Austria, Switzerland, the Netherlands, Ireland, Malaysia, New Zealand, and Singapore. A Firefox Account is required to use the Service.
```

Comment:
Should we list the countries in alphabetical order, even if it is within a region/continent?

Reply:
Yes. It would make sense to do that, but I'll make that a universal change across all the terms of service. later on.